plugins/modules/foreman_architecture.py

```diff
@@ -105,7 +107,7 @@ def main():
 
     entity = module.find_resource_by_name('architectures', name=entity_dict['name'], failsafe=True)
 
-    changed = module.ensure_resource_state('architectures', entity_dict, entity)
+    changed, _ = module.ensure_entity('architectures', entity_dict, entity)
```

Comment:
do we want to have a shortcut function for that, that only returns `changed`? like `ensure_entity_state`?

Reply:
I guess, i was too lazy. Will add it.

Reply:
On the other hand i dream of always returning the changed entity from the module.

Reply:
I have a "implement diff mode for all apypie modules" todo entry, so at some point... but until then the shortcut is handy :)

Reply:
Ahh, that is why you are interested in the `changed_entity`. ;)